docroot/modules/custom/civictheme_content/civictheme_content.install

```diff
@@ -62,16 +63,119 @@ function civictheme_content_install() {
     return 'civictheme_content_' . $value;
   }, $supported);
 
+  $theme_block_list = [
```

Comment:
It would be better to have this as a function in `civictheme.provision.inc` called `civictheme_get_theme_block` which returns a map as follows:

```
[
<banner_key> : [

'region' => '...',
'uuid' => '...',
'builder' => 'civictheme_provision__<type>'

...
]


Then include this file into the install and use this data structure to control deletion.

Then update provision to use this data structure to drive the building of the paragraphs as well - that way all the data required for these operations are controlled in one structure that we can add and modify easily.